kiali.go

```diff
@@ -1,3 +1,18 @@
+// Kiali
+//
+// Kiali project, observability for the Istio service mesh
+//
+//     Schemes: http
+//     BasePath: /api
+//     Version: 0.2.0.Alpha
```

Comment:
We will need a good way to update this on releases.
It will be pretty easy to not put the correct version here.
Perhaps this needs a comment in the README or in the releases doc.

Reply:
cc @jmazzitelli

Reply:
+1 This version only set this information in https://kiali.org/pages/api/overview.html 

Version information

Version : 0.2.0.Alpha